cmd/scollector/collectors/httpunit.go

```diff
@@ -46,16 +48,32 @@ func cHTTPUnit(plans *httpunit.Plans) (opentsdb.MultiDataPoint, error) {
 	unix_now := time.Now().Unix()
 	var md opentsdb.MultiDataPoint
 	for r := range ch {
+		if r == nil || r.Case == nil || r.Plan == nil {
+			slog.Errorf("httpunit: unexpected test plan on channel, the plan result, the case, and/or the plan result is missing.")
+			continue
+		}
 		tags := opentsdb.TagSet{
 			"protocol":     r.Case.URL.Scheme,
 			"ip":           r.Case.IP.String(),
 			"url_host":     r.Case.URL.Host,
 			"hc_test_case": r.Plan.Label,
 		}
+		metadata.AddMeta("", opentsdb.TagSet{"hc_test_case": r.Plan.Label}, "url", r.Case.URL.String(), false)
+		if r.Result == nil || r.Result.Result != nil {
+			Add(&md, "hu.error", true, tags, metadata.Gauge, metadata.Bool, descHTTPUnitError)
```

Comment:
Is there a better name for this metric than `hu.error`? Seems unintuitive what 0/1 mean. Maybe its just me.

Reply:
Also no metadata.

Reply:
Existing metric I'm not looking to rename at the moment.

Reply:
I'm not seeing what you referencing with "no metadata?"

Reply:
nvm me. I forget how things work.